tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml

```diff
@@ -51,7 +51,6 @@
      <standardCssUrl>tds.css</standardCssUrl>
      <catalogCssUrl>tdsCat.css</catalogCssUrl>
      <datasetCssUrl>tdsDst.css</datasetCssUrl>
-     <openDapCssUrl>tdsDap.css</openDapCssUrl>

```

Comment:
Even though OPeNDAP has it's own stylesheet, there's no hook for user-supplied yet